server/app/services/call_service.py

```
@@ -1208,10 +1240,22 @@ async def list_calls_by_status(
         return list(result.scalars().all())
 
+    # Supported country codes with their validation rules
+    # Format: country_code -> (code_string, required_digits, country_name)
+    SUPPORTED_COUNTRY_CODES = {
+        '1': ('1', 10, 'US'),
+        '91': ('91', 10, 'India'),
+        # Add more countries here as needed:
+        # '44': ('44', 10, 'UK'),
+        # '61': ('61', 9, 'Australia'),
+        # '86': ('86', 11, 'China'),
+    }
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)